### PR TITLE
Add Common App

### PR DESCRIPTION
**Pull request by hkamran80**

### CODE REVIEW

File: entries/c/commonapp.org.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "Common App": {
+    "domain": "commonapp.org",
+    "contact": {
+      "facebook": "commonapp",
+      "twitter": "CommonApp"
+    },
+    "keywords": ["education"]
```
Comment (Carlgo11) [inline entries/c/commonapp.org.json]:
```suggestion
    "keywords": [
      "universities"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Whilst universities around the world are listed, the service seems to be for U.S. residents.
I'd therefore recommend a regions array.

File: img/c/commonapp.org.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...8 4 2.2 5.3s3.2 2.2 5.3 2.2c4.4.1 7.4-3.4 7.4-7.5z"/><linearGradient id="a" x1="483.17" x2="506.031" y1="379.774" y2="250.119" gradientUnits="userSpaceOnUse"><stop offset=".1" style="stop-color:#338ede"/><stop offset=".235" style="stop-color:#37a3e6"/><stop offset=".514" style="stop-color:#42d9f9"/><stop offset=".523" style="stop-color:#42dbfa"/><stop offset=".622" style="stop-color:#4bddfb"/><stop offset=".788" style="stop-color:#65e1fc"/><stop offset=".998" style="stop-color:#8fe9ff"/><stop offset="1" style="stop-color:#8fe9ff"/></linearGradient><path d="M472.7 336.4c-4.9-2.6-7-6.4-6.8-12.2.1-3.9 3.7-7.1 3.9-7.2l.5-.4 31.5 15.1c2.2 1.1 3.7 3.4 3.7 5.9V352l-32.8-15.6zm-3.2-29.3s-3.7-3.1-3.7-8.3c0-2.8.8-5.1 2.5-7.2 1.5-1.9 3.8-3.6 6.7-5 3.8-1.9 41.2-19.4 58.7-27.6v14.1c0 2.7-1.6 5.2-4 6.4l-59.6 28.1-.6-.5zm2.3-27.1c-8.7 4.3-13.3 10.9-13.3 18.9 0 6.9 3.8 11.5 5.6 13.3-1.8 1.7-5.5 6.1-5.6 11.9-.3 8.5 3.5 15.1 10.9 19l40.6 19.3c.2.1.5.2.8.2.3 0 .7-.1 1-.3.5-.3.8-.9.8-1.6v-23.1c0-5.3-3.1-10.2-7.9-12.5l-27-12.9 55-26c5-2.4 8.2-7.5 8.2-13v-22.4c0-.4 0-1.5-.8-2.1-.3-.2-.5-.2-.8-.2-.4 0-.8.2-1 .3-6.2 2.8-61.9 28.9-66.5 31.2z" style="fill-rule:evenodd;clip-rule:evenodd;fill:url(#a)"/></svg>
```
Comment (Carlgo11) [inline img/c/commonapp.org.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 82.42 114.1"><linearGradient id="a" x1="24.69" x2="47.55" y1="131.27" y2="1.62" gradientUnits="userSpaceOnUse"><stop offset=".1" stop-color="#338ede"/><stop offset=".23" stop-color="#37a3e6"/><stop offset=".51" stop-color="#42d9f9"/><stop offset=".52" stop-color="#42dbfa"/><stop offset=".62" stop-color="#4bddfb"/><stop offset=".79" stop-color="#65e1fc"/><stop offset="1" stop-color="#8fe9ff"/><stop offset="1" stop-color="#8fe9ff"/></linearGradient><path fill="url(#a)" d="M14.22 87.9c-4.9-2.6-7-6.4-6.8-12.2.1-3.9 3.7-7.1 3.9-7.2l.5-.4 31.5 15.1c2.2 1.1 3.7 3.4 3.7 5.9v14.4zm-3.2-29.3s-3.7-3.1-3.7-8.3c0-2.8.8-5.1 2.5-7.2 1.5-1.9 3.8-3.6 6.7-5 3.8-1.9 41.2-19.4 58.7-27.6v14.1c0 2.7-1.6 5.2-4 6.4l-59.6 28.1zm2.3-27.1C4.62 35.8.02 42.4.02 50.4a19 19 0 0 0 5.6 13.3c-1.8 1.7-5.5 6.1-5.6 11.9-.3 8.5 3.5 15.1 10.9 19l40.6 19.3c.2.1.5.2.8.2.3 0 .7-.1 1-.3.5-.3.8-.9.8-1.6V89.1c0-5.3-3.1-10.2-7.9-12.5l-27-12.9 55-26c5-2.4 8.2-7.5 8.2-13V2.3c0-.4 0-1.5-.8-2.1-.3-.2-.5-.2-.8-.2-.4 0-.8.2-1 .3-6.2 2.8-61.9 28.9-66.5 31.2z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/c/commonapp.org.json (hunk 1 above)
Comment (hkamran80) [reply entries/c/commonapp.org.json:8]: It's not just for U.S. residents, [read more here](https://www.commonapp.org/static/88498a0ee6d39ba4540d8e658c113cb4/FYInternationalCAReady.pdf).

**Comment by hkamran80** [comment]
> Whilst universities around the world are listed, the service seems to be for U.S. residents. I'd therefore recommend a regions array.

It's not just for U.S. residents, however, it's primarily for U.S. universities. You can find out more [here](https://www.commonapp.org/static/88498a0ee6d39ba4540d8e658c113cb4/FYInternationalCAReady.pdf).

**Comment by Carlgo11** [comment]
Still, a regions array would be good.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**